spec/memoize_query_spec.rb

```diff
@@ -443,11 +443,53 @@ def expect_to_eq_with_or_without_redis
     end
   end
 
-  it 'does not memoize queries with NOT' do
-    expect_not_to_use_redis do
-      Site.where(id: Site.where.not(a: 1)).to_a
+  context 'when the queries have NOT' do 
+    let!(:record) { Site.create!(a: 1, b: 1) }
+    let!(:relation_with_only_not) { Site.where.not(a: 2) }
```

Comment:
Is there anything special to multiple not conditions such as `.not(a: 1, b: 2)`?

Reply:
yeah sure, added